llvm/test/CodeGen/RISCV/rvv/vl-opt-op-info.mir

```diff
@@ -602,4 +602,83 @@ body: |
     %x:vmv0 = PseudoVMAND_MM_M1 $noreg, $noreg, -1, 0
     %y:vrnov0 = PseudoVADD_VV_MF2_MASK $noreg, $noreg, $noreg, %x, 1, 3 /* e8 */, 0
 ...
-
+---
+name: vmop_vv
+body: |
+  bb.0:
+    ; CHECK-LABEL: name: vmop_vv
+    ; CHECK: %x:vr = PseudoVMSEQ_VV_M1 $noreg, $noreg, 1, 0 /* e8 */
+    ; CHECK-NEXT: %y:vr = PseudoVMAND_MM_M1 $noreg, %x, 1, 0 /* e8 */
+    %x:vr = PseudoVMSEQ_VV_M1 $noreg, $noreg, -1, 0
```

Comment:
The EEW for a VMSEQ must be 3, 4, 5, or 6.

Reply:
Thanks. Updated. Is that something we could enforce in the MachineVerifier?